app/src/main/java/com/kickstarter/viewmodels/MessagesViewModel.java

```diff
@@ -39,14 +46,20 @@
     /** Emits a boolean that determines if the backing info view should be hidden. */
     Observable<Boolean> backingInfoViewHidden();
 
-    /** Emits the participant name to be displayed. */
-    Observable<String> participantNameTextViewText();
-
     /** Emits a list of messages to be displayed. */
     Observable<List<Message>> messages();
 
+    /** Emits the participant name to be displayed. */
+    Observable<String> participantNameTextViewText();
+
     /** Emits the project name to be displayed. */
     Observable<String> projectNameTextViewText();
+
+    /** Emits an empty string when we want to clear the message edit text. */
+    Observable<String> setEmptyMessageEditText();
```

Comment:
seems we could just call this `setMessageEditText` and not even communicate that we are only emitting empty strings. the view doesn't need to know that, it just needs to follow orders!